### ISSUE TITLE
MEDGENCUI possible future edge case handling

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/medgen#20]
Regarding contingent MEDGENCUI preservation

Options discussed:
a. Test that fails if any MEDGENCUI appears
b. Let them slip in with the rest of the UMLS / MEDGEN.
c. Filter them out. Write them to a separate file. Or log to a file.
d. A GH action that reviews the release and detects if MEDGENCUI and throw warning.

_Edit_: This only applies to the Mondo robot template, not for the HPO mappings work: #18.

**Comment by joeflack4** [comment]
@twhetzel We discussed this at meeting today. But I also saw another comment from Nico [in another PR](https://github.com/monarch-initiative/medgen/pull/19#discussion_r1551401726). It looks like both you and Nico concur in the importance of handling this.

I'm going to go with option (c). And it will not take me long. So I will do it now

_Edit_: Actually went with an alternative to (c). Added a parameter with default to True to filter them out, and just filtered them out.
